wiki/History_of_osu!/2015/en.md

```diff
@@ -1,39 +1,185 @@
+---
+tags:
+  - history
+  - 2015
+  - legacy
+---
+
 # History of osu! 2015
 
+## February
+
+On 16 February 2015, storyboard rules were updated to allow image sizes larger than 854x640. As  suggested by [Xgor](https://osu.ppy.sh/users/98661), the new maximum rankable image size for storyboards was updated to 1366x768 [[1,2]][r].
+
 ## March
 
-peppy reconsiders that his current business model will not obtain him his "Private osu! G6 Jet" and enables the *[osu!coin](/wiki/osu!coin)* to hopefully increase revenue. Every current player was given 10 osu!coins to play (at the rate of 1 play = 1 osu!coin). These can be replenished after obtaining a combo of multiples of 100. Or simply by purchasing them at these rates:
+The first ever edition of osu!weekly was released on 13 March 2015. Written by [Tasha](https://osu.ppy.sh/users/1031958), osu!weekly positioned itself as "a weekly news digest covering the latest happenings of the game and its community" [[2]][r].
 
-- 1 osu!coin = $1 USD
-- 10 osu!coins = $10 USD
-- 1000 osu!coins = $999USD (Saves you 0.001%! BEST VALUE!)
-- Infinite osu!coins = Your first unborn child - osu! needs more team members, have your unborn child work off your debts! :)
+The first ever osu! Beatmap Blueprint stickers were added to the [osu!store](https://osu.ppy.sh/store/listing) on 20 March 2015. The stickers came in two packs of 8.5" x 11" custom sticker sheets featuring such elements as hit circles, combo counters, sliders, and much more [[3]][r]. In celebration of the launch, a contest was run to see who could make the best [beatmap](/wiki/Beatmap) using the stickers, with a first place prize of an osu! tablet and osu! sticker sets 1 and 2. Despite this, the contest was cancelled due to a lack of submissions, and no follow-up regarding this was ever publicly stated at the time [[3,4]][r]. <!--price needed-->
 
-Links:
+[Wii Remote](https://en.wikipedia.org/wiki/Wii_Remote "Wikipedia") support was added for [osu!taiko](/wiki/Game_mode/osu!taiko) on 26 March 2015 [[5]][r]. <!--changelog entry missing-->
 
-- [osu!coins](https://osu.ppy.sh/home/news/2015-03-31-osucoins)
+On 31 March 2015, [osu!coins](/wiki/osu!coins) were added as a joke in-game currency for osu! as a part of an April Fools' day joke. The coins were made to parody [free-to-play](https://en.wikipedia.org/wiki/Free-to-play "Wikipedia") mobile games' common use of in-game currencies and [microtransactions](https://en.wikipedia.org/wiki/Microtransaction "Wikipedia"). The osu!coins were quickly removed the following day.
```

Comment:
Shall we link to https://osu.ppy.sh/wiki/en/History_of_osu%21/April_Fools#2015

Reply:
Oh, yes, we should. At the time of writing the April Fools' pr was still in review, which is why it wasn't originally linked

Reply:
> Oh, yes, we should. At the time of writing the April Fools' pr was still in review, which is why it wasn't originally linked

Should be https://osu.ppy.sh/wiki/zh/History_of_osu%21/April_Fools#2015 
or https://osu.ppy.sh/wiki/zh/osu%21coin
https://osu.ppy.sh/wiki/zh/History_of_osu%21/April_Fools#osu!coin is invalid

Reply:
the Chinese version (zh.md) is invalid because there is no Chinese translation for this article. `/wiki/History_of_osu!/April_Fools#osu!coins` should redirect accordingly